package.json

```diff
@@ -2,18 +2,22 @@
   "name": "@ngrx/store",
   "version": "2.1.2",
   "description": "RxJS powered Redux for Angular2 apps",
-  "main": "./index.js",
+  "main": "./bundles/store.umd.js",
+  "module": "./index.js",
   "scripts": {
-    "clean": "rimraf node_modules typings dist tmp npm-debug.log && npm cache clean",
-    "clean:test": "rimraf tmp",
-    "clean:dist": "rimraf dist",
-    "build:dist": "npm run clean:dist && ngc && cp package.json dist/package.json",
-    "lint": "tslint src/**.ts",
-    "test": "tsc -p spec && jasmine && npm run clean:test",
-    "test:ngc": "ngc -p spec/tsconfig.ngc.json",
-    "typings": "typings install",
-    "prebuild:dist": "npm run typings",
-    "prepublish:dist": "npm run build:dist"
+    "karma": "karma start --single-run",
+    "test:unit": "npm run karma",
+    "test:ngc": "ngc -p ./spec/ngc/tsconfig.ngc.json",
+    "test": "npm run test:unit && npm run test:ngc",
```

Comment:
I believe in one of the adjacent projects ("core"?) I saw npm-run-all being used for this purpose. I don't know if one approach of the other is considered better, though I do suggest picking one approach and using it for this  family of related projects.